src/sales/cart.md

```diff
@@ -8,3 +8,26 @@ The cart is positioned at the end of the path to purchase, at the intersection o
 
 ![The shopping cart page displays tools the shopper can use to manage the products for their order ]({% link images/images/storefront-cart-full.png %}){: .zoom}
 _Example storefront - shopping cart_
+
+## How to configure the Minimum Qty Allowed in Shopping Cart
+
+The admin can set the minimum qty allowed to order in store. In addition, the minimum quantity admin can configure for each customer group.
+
+1. Go to **Stores** > **Settings** > **Configuration**.
+
+1. Select **Inventory** under **Catalog** section.
+
+1. Expand **Product Stock Options** section.
+
+1. Set the minimum quantity in the **Minimum Qty Allowed in Shopping Cart** field.
+
+    To edit this field clear the **Use system value** checkboxes.
+
+    - Select Customer Group for setting minimum quantity limit for specific groups. To add another condition click **Add Minimum Qty**.
+
+    - To set a minimum quantity limit for all customers select **ALL GROUPS**.
+
+1. Click **Save Config**.
+
+    ![]({% link images/images/minimum-qty-allowed-in-shopping-cart.png %}){: .zoom}
+    _Minimum Qty Allowed to Order_
```

Comment:
```suggestion
    _Minimum quantity requirement in shopping cart_
```

Reply:
Done, a new topic is created, please check🙏